src/core/handshaker/http_connect/http_proxy_mapper.cc

```diff
@@ -192,9 +191,9 @@ absl::optional<grpc_resolved_address> GetAddressProxyServer(
   }
   auto address = StringToSockaddr(*address_value);
   if (!address.ok()) {
-    gpr_log(GPR_ERROR, "cannot parse value of '%s' env var. Error: %s",
-            HttpProxyMapper::kAddressProxyEnvVar,
-            address.status().ToString().c_str());
+    LOG(ERROR) << "cannot parse value of '"
+               << std::string(HttpProxyMapper::kAddressProxyEnvVar)
+               << "' env var. Error: " << address.status().ToString();
     return absl::nullopt;
```

Comment:
Converting to std::string is needed here coz otherwise it will break the build.
HttpProxyMapper::kAddressProxyEnvVar is constexpr.